crates/exex/exex/src/notification.rs

```diff
@@ -22,6 +23,8 @@ pub enum ExExNotification {
         /// The old chain before reversion.
         old: Arc<Chain>,
     },
+    /// Finalized block header.
+    FinalizedBlock(Option<SealedHeader>),
```

Comment:
why is it `Option` and not just a `SealedHeader`?

Reply:
you're right it should be without Option.